### DISCUSSION TITLE
Set duration of comp to length of imported footage

### DISCUSSION THREAD

**Discussion by nickgwyer** [inlife/nexrender#895] [General]
Hi,

Is it possible to set the render length of a comp to the length of an imported piece of footage please? I would like to be able to import footage of different lengths into pre exisiting comps and trim the length of the comps to the length of the imported footage video and render it.

Is that possible with nextrender and jsx code?

Many thanks.

Nick

**Comment by inlife** [comment]
It is. You can change the `duration` property on `layer.source`. Alternatively, for non-asset layers, there are also an inPoint/outPoint properties that can be changed.
